app/components/simple-paginator/template.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if hasPages}}
+    <span class='SimplePaginator__element'>
+        {{#bs-button disabled=(not hasPrev) onClick=(action previousPage)}}
+            <span aria-label="{{t 'paginator.previous'}}">
+                {{fa-icon 'chevron-left'}}
+            </span>
+        {{/bs-button}}
+    </span>
+    <span class='SimplePaginator__element'>
+        {{curPage}}
```

Comment:
It's not very clear these are page controls: 
<img width="128" alt="screen shot 2018-05-02 at 16 10 57" src="https://user-images.githubusercontent.com/6776190/39546827-b06ba84e-4e23-11e8-9c6b-b5667f1134cd.png">

I think it'd be better if it said "Page 1 of 4" or something along those lines.